packages/mdc-menu/README.md

```diff
@@ -150,6 +153,7 @@ Property | Value Type | Description
 `open` | Boolean | Proxies to the foundation's `isOpen`/(`open`, `close`) methods. 
 `items` | Array<Element> | Proxies to the foundation's container to query for all `.mdc-list-item[role]` elements.
 `itemsContainer` | Element | Queries the foundation's root element for the `mdc-menu__items` container element.
+`quickOpen` | Boolean | Proxies to the foundation's `quickOpen()` method. 
```

Comment:
Description needs updating to `setQuickOpen()` now

Reply:
It also occurs to me that `setQuickOpen` needs to be documented under the Foundation API... and make sure to mention that it affects both open _and_ close animations. (Which makes me think maybe this isn't the best name, but not sure what is... maybe flip it to `enableAnimations`, but maybe that's too vague?)